src/main/notifications/index.js

```diff
@@ -24,9 +26,20 @@ export function displayMention(title, body, channel, teamId, silent, webcontents
         silent,
         data,
     };
-    const mention = new Mention(options);
+
+    const mention = new Mention(options, channel);
 
     mention.on('show', () => {
+        // On Windows, manually dismiss notifications from the same channel and only show the latest one
+        if (process.platform === 'win32') {
+            for (const notification of currentNotifications.values()) {
+                if (notification.channel.id === mention.channel.id) {
+                    notification.close();
+                    currentNotifications.delete(notification);
```

Comment:
Do we need to do anything to remove notifications from `currentNotifications` when they're dismissed?

Reply:
Isn't that what `currentNotifications.delete` is doing? Should be removing it from the `Set`. Do you mean destroying the object?

Reply:
I mean, is there anything to do if the notification is dismissed by the user before this triggers? I think that's the same thing Guillermo is asking about as well.

Reply:
Nope, if it's dismissed then `close()` just doesn't do anything. In terms of the `Set` we always just get rid of the last notification regardless.